application/controllers/RedundancygroupController.php

```diff
@@ -0,0 +1,332 @@
+<?php
+
+/* Icinga DB Web | (c) 2024 Icinga GmbH | GPLv2 */
+
+namespace Icinga\Module\Icingadb\Controllers;
+
+use Icinga\Exception\NotFoundError;
+use Icinga\Module\Icingadb\Common\CommandActions;
+use Icinga\Module\Icingadb\Common\Links;
+use Icinga\Module\Icingadb\Data\Nodes;
+use Icinga\Module\Icingadb\Model\DependencyNode;
+use Icinga\Module\Icingadb\Model\RedundancyGroup;
+use Icinga\Module\Icingadb\Model\RedundancyGroupSummary;
+use Icinga\Module\Icingadb\Web\Control\SearchBar\ObjectSuggestions;
+use Icinga\Module\Icingadb\Web\Control\ViewModeSwitcher;
+use Icinga\Module\Icingadb\Web\Controller;
+use Icinga\Module\Icingadb\Widget\DependencyNodeStatistics;
+use Icinga\Module\Icingadb\Widget\Detail\MultiselectQuickActions;
+use Icinga\Module\Icingadb\Widget\Detail\RedundancyGroupDetail;
+use Icinga\Module\Icingadb\Widget\ItemList\DependencyNodeList;
+use ipl\Html\HtmlElement;
+use ipl\Html\Text;
+use ipl\Orm\Query;
+use ipl\Stdlib\Filter;
+use ipl\Web\Control\LimitControl;
+use ipl\Web\Control\SortControl;
+use ipl\Web\Url;
+use ipl\Web\Widget\Tabs;
+
+class RedundancygroupController extends Controller
+{
+    use CommandActions;
```

Comment:
override the process check result action, it won't work after all and isn't provided in the quick action bar